pages/background.md

```diff
@@ -1,25 +1,30 @@
+{% include nav.html %} {% include css.html %}
+
 # Project Background
 
 ---
 
-Put a general discussion of your topic here. Don't get into the historiography or scholarship here. Just introduce to your reader what your topic is. What is the 'problem', the 'question', and/or the 'argument'.
+The Triangular trade was by far one of the worlds most important trade route. The route funnelled the foundations of slave labor into much of the carribean and the southern United States. However, has the slave trade ever been infiltrated by acts of piracy, and to what extent did piracy hinder the progression of the Triangular trade? With our thesis in tow, we aim to understand the economic effect piracy had on Slave Voyages.
```

Comment:
Okay, good. But why not throw some links/citations here for some of the main authors who talk about the triangle trade whom you've found? Even if you don't mention them by name, putting in a bibliographical reference is important normally, and doubly so on something like a background page. Digital projects usually include references to scholars when doing a detail background like this.